scss/_patterns_links.scss

```diff
@@ -11,6 +11,36 @@
     &--no-underline {
       border: 0;
     }
+
+    &--soft {
+      color: $color-dark;
+    }
+
+    &--strong {
+      border-bottom-color: $color-highlight;
+      color: $color-dark;
+      font-weight: 400;
+
+      &:hover {
+        color: $color-highlight;
+      }
+    }
+
+    &--inverted {
+      border-bottom-color: $color-light;
+      color: $color-light;
+      font-weight: 400;
+
+      &:hover {
+        color: $color-light;
+        border-bottom-color: $color-light;
+      }
+
+      &:visited {
+        color: darken($color-light, 10%);
```

Comment:
This colour change is so subtle. It is almost impossible to tell the difference.